userspace/falco/grpc_context.h

```diff
@@ -36,7 +36,7 @@ class context
 {
 public:
 	context(::grpc::ServerContext* ctx);
-	~context() = default;
+	virtual ~context() = default;
```

Comment:
Same here with context, why virtual now? and what does `context` mean to you?

Reply:
The rationale is: `context` is a base class and instances of the derived classes get destructed.

When a base class destructor is not virtual and you have a variable pointing to an object derived from the base class, deleting the derived instance has undefined behaviour. Which can lead to memory leaks.